Application/global_settings_manager.py

```diff
@@ -68,6 +68,17 @@ def load_global_settings(self):
         settings = QSettings()
 
         settings.beginGroup("global-settings")
+
+        settings.beginGroup("user-settings")  # open user settings
```

Comment:
The comment "# open user settings" is not accurate to what this line of code actually does. It would be best to remove it.

QSettings simply stores key-value pairs (similar to a hashmap). Groups are a hierarchical naming format to the keys. For instance, the key "table_padding" is actually "global-settings/user-settings/table_padding". It's a string like any other key.  We do not actually "open" anything as we enter and leave groups, but instead are adding or removing a suffix to the key we're accessing or writing to.